### ISSUE TITLE
Any way to stream large response?

### ISSUE THREAD

**Issue by madprogrammer** [twisted/klein#409]
Is there any way to stream a large response? I need to return a large JSON array, and I want to do it without loading all the data into memory.

**Comment by twm** [comment]
Yes, you can return a [`Resource`](https://twistedmatrix.com/documents/current/api/twisted.web.resource.Resource.html) that streams by calling `request.write(b'...'); request.write(b'...'); request.finish()` in the usual Twisted Web way.
